pkg/config/editor_presets.go

```diff
@@ -44,6 +44,7 @@ func getPreset(osConfig *OSConfig, guessDefaultEditor func() string) *editPreset
 		"emacs":   standardTerminalEditorPreset("emacs"),
 		"nano":    standardTerminalEditorPreset("nano"),
 		"kakoune": standardTerminalEditorPreset("kakoune"),
+		"hx":      helixEditorPreset(),
```

Comment:
Shouldn't the name of the preset be `helix` rather than `hx`? That's what `Config.md` above says, and I'd prefer the more descriptive name (like with "code" vs. "vscode"). We'll then need an entry `"hx": "helix",` in the `editorToPreset` map below.

Reply:
Indeed, this one bit me!